data/human/deep missions.txt

```diff
@@ -519,7 +519,7 @@ mission "Deep: Mystery Cubes 3"
 			`	One of the Navy crewmen produces a football from their cabin, and a small game begins inside of an open landing pad. Others try to pass the time by reading or writing. Everyone comes to a standstill though once the ground begins to rumble; an earthquake. A Navy member calls out to everyone. "Don't worry, this is a small one. We get earthquakes like this a lot on Gemstone." The shaking eventually subsides, and you decide to go into your ship to rest.`
 			
 			label mirfak
-			`	Suddenly, your communication device beeps and you hear Lieutenant Paris' voice, "The blockade in Mirfak has just been attacked. I repeat, the blockade in Mirfak has just been attacked. Reports are that the blockade has sustained heavy losses as the pirate fleet included one of the stolen Cruisers."`
+			`	Suddenly, your communication device beeps and you hear Lieutenant Paris's voice, "The blockade in Mirfak has just been attacked. I repeat, the blockade in Mirfak has just been attacked. Reports are that the blockade has sustained heavy losses as the pirate fleet included one of the stolen Cruisers."`
```

Comment:
I will once again express my preference for the former.

Reply:
Fixed, but it should be noted that MZ's writing uses "s's".

free worlds reconciliation.txt: `You find Ijs's office; Katya knocks and says, "Professor Springborn, are you having office hours right now?"`

Reply:
Aside from the instances in this file, here's what I'm able to find on `s's` in the game currently.
* human\free worlds reconciliation.txt:	`...You find Ijs's office...`
* human\hails.txt: `"What did they change Uranus's name to? Urass!"`
* intro missions.txt: `...The expression on James's face is growing increasingly concerned as she speaks...`
* tooltips.txt: `When firing this weapon then the ships's heat increases...`
* tooltips.txt: `... Ionization reduces the ships's energy and slowly wears off over time...`
  * The above two are definitely errors and should just be `ship's`.

And again, aside from this file, here's what I was able to find for `s'` at the end of names.
* coalition\coalition missions.txt: `... In my case, here to thank her for Adfices' life...`
* hai\hai news.txt: `Matthias' Sky Shuttles is the perfect service for anyone making trips to, from, and within Hai space. Fly 'Hai' into the sky with Matthias' Sky Shuttles!`
* human\hails.txt: `get sent to Davy Jones' locker!`
* human\names.txt: `Davy Jones' Locker`
* remnant\remnant 1 introduction.txt: `True to Aeolus' word, there are crates of Hai quantum keystones waiting for you...`
* remnant\remnant 2 cognizance.txt: `You have destroyed the Korath ship that Taely sent you to hunt down. Return to <destination> to report the Mergansers' performance.`

So in either case looks like we'd need to swap some stuff over. There seems to be no definitive answer as to how to create the possessive form of names ending in s; I see that places like the Associated Press suggest `s'` while the Chicago Manual of Style suggests `s's`. Just seems like something we'd need to decide on.

Reply:
I also prefer the former s' method.

Reply:
Yeah, I couldn't find anything definitive either. Both are acceptable, and there's no tendencies in American vs British English as far as I can tell. I've got no strong preferences, so up to y'all.

Reply:
If it adds anything, Australian standards come down exclusively on the side of `s'` and reject `s's`